src/frequenz/channels/util/_timer.py

```diff
@@ -270,36 +270,49 @@ def __repr__(self) -> str:
 class Timer(Receiver[timedelta]):
     """A timer receiver that triggers every `interval` time.
 
-    The timer as microseconds resolution, so the `interval` must be at least
+    The timer as microseconds resolution, so the
+    [`interval`][frequenz.channels.util.Timer.interval] must be at least
     1 microsecond.
 
-    The message it produces is a `timedelta` containing the drift of the timer,
-    i.e. the difference between when the timer should have triggered and the time
-    when it actually triggered.
+    The message it produces is a [`timedelta`][datetime.timedelta] containing the drift
+    of the timer, i.e. the difference between when the timer should have triggered and
+    the time when it actually triggered.
 
     This drift will likely never be `0`, because if there is a task that is
     running when it should trigger, the timer will be delayed. In this case the
     drift will be positive. A negative drift should be technically impossible,
-    as the timer uses `asyncio`s loop monotonic clock.
+    as the timer uses [`asyncio`][asyncio]s loop monotonic clock.
```

Comment:
We seem to be using a mix of of `()` and `[]` here. Is `[name][ref]` different from `(name)[ref]`?

Reply:
`[name](ref)` is a regular, standard, markdown link, `[name][ref]` is a `mkdocstrings` extension to do code-crosslinking, so `ref` can only be a valid fully-qualified Python symbol.

Reply:
When configuring [external "object inventories"](https://mkdocstrings.github.io/python/usage/#import), this references can even point to external python packages, in this case `asyncio` will link to Python's official docs.

Reply:
BTW, `(name)[ref]` is incorrect, so if you spotted one is a bug, but I'm assuming is just a typo in your GitHub comment.

Reply:
This is one of my forever confusions in markdown syntax :sweat_smile: I can never remember the right order